### DISCUSSION TITLE
IsometricTileMapComponent from .tmx

### DISCUSSION THREAD

**Discussion by Pingear** [flame-engine/flame#1357] [Q&A] [answered]
Hey)
The documentation (https://docs.flame-engine.org/1.0.0/tiled.html ) says that Tiled(https://www.mapeditor.org/) is an great tool to design levels and maps.
But I don't see how I can create IsometricTileMapComponent from *.tmx file. Moreover, IsometricTileMapComponent is not part of the flame_tiled package at all and is not documented. Can you please tell me how to import isometric maps created in mapeditor? Or did you not develop such functionality? Maybe some advice?

**Comment by spydon** [comment, marked as answer]
Hi, there is no such functionality currently unfortunately.
You can have a look [here](https://github.com/bluefireteam/heeve) for an example of how to use the `IsometricTileMapComponent`, there are separate docs for it [here](https://github.com/bluefireteam/heeve) too.

**Reply by Pingear** [reply]
thanks a lot)
Cool, I was just looking for a project like this!!

**Reply by Pingear** [reply]
@spydon 
Tell me please. You use .fa files in your [sprites](https://github.com/bluefireteam/heeve/tree/main/assets/sprites), what are those files? I tried to google, but could not find information about files with this extension (

**Reply by spydon** [reply]
It is FireAtlas files: 
https://fire-atlas.flame-engine.org/
https://github.com/flame-engine/fire-atlas/
